src/lib.rs

```diff
@@ -26,3 +26,45 @@ pub mod datastore;
 pub mod query;
 pub mod endpoints;
```

Comment:
Would be cool if you could add a library feature named "http" or something so we can conditionally remove all rocket stuff at compile time.

Or maybe that will be hard since we use rocket_contrib Json,JsonValue on a lot of places? (i assume that rocket_contrib depends on rocket).

Reply:
One day maybe, but right now I just want to get this done. Works fine regardless.